Question and answer activity

amitsagtani97 opened the pull request:
Todo:
Fix layouts for vertical mode.
Create admin mode.
Task -- > [https://phabricator.kde.org/T5469](https://phabricator.kde.org/T5469)

petitlapin commented on src/activities/question_and_answer/ActivityInfo.qml:
```diff
@@ -0,0 +1,40 @@
+  title: "Question_and_answer activity"
+  //: Help title
+  description: ""
+  //intro: "put here in comment the text for the intro voice"
```
need to be updated

AkshayCHD commented:
@amitsagtani97  Hii, can i complete this activity, it would be helpful if you could just let me know what are the things left to implement here.

amitsagtani97 commented:
@AkshayCHD Hi, sure you can complete it.
Here is the link to the task -- https://phabricator.kde.org/T5469
Here is the link to the google doc, which I proposed for this activity -- https://docs.google.com/document/d/1tzA5r0nsBfPoBjYl81a0qTsVt6aHnP_KfcB1CmPIrfE/edit?usp=sharing

I have implemented the Menu Screen and added two sub-activities.
The overall layout needs to be improved for vertical screens and an admin mode needs to be implemented as discussed in the proposal.
Feel free to ask in case of any doubt :)

AkshayCHD commented:
@amitsagtani97  hii, i wanted to ask, in admin mode we have to allow the user to choose the questions on his own, right?

amitsagtani97 commented:
@AkshayCHD In admin mode, we can select the questions to be asked.
This allows tutors to create different sets of questions for the kids to solve and practice.
